nixos/modules/i18n/input-method/default.nix

```diff
@@ -29,7 +29,7 @@ in
   options.i18n = {
     inputMethod = {
       enabled = mkOption {
-        type    = types.nullOr (types.enum [ "ibus" "fcitx" "nabi" "uim" "hime" ]);
+        type    = types.nullOr (types.enum [ "ibus" "fcitx" "fcitx5" "nabi" "uim" "hime" ]);
```

Comment:
In the original issue,
> Keep it for now but mark it as deprecated with a note about fcitx5? 

This can be done in `aliases.nix`, and remove fcitx in the future.

Reply:
fcitx is still being maintained and people are still using it so I see no point in deprecating it.